src/App.jsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import Home from "./containers/Home";
 import Footer from "./components/Footer";
+import Form from "./containers/Form";
```

Comment:
Can you give the form component a more meaningful name, i.e. DonateFrom maybe or whatever you find more convenient

Reply:
PTAL